tests/utils.py

```diff
@@ -182,8 +182,7 @@ def setup_test_homeserver(
     If no datastore is supplied, one is created and given to the homeserver.
 
     Args:
-        cleanup_func : The function used to register a cleanup routine for
-                       after the test.
+        self: The UnitTest class.
```

Comment:
I think it needs to be a TestCase (instance) rather than a UnitTest (class)?

```suggestion
        self (twisted.trial.unittest.TestCase): The test instance
```